src/main/java/frc/robot/Robot.java

```diff
@@ -23,6 +20,7 @@
     public PowerDistribution pdp = new PowerDistribution();
     private RobotContainer m_robotContainer;
     private Command m_autonomousCommand;
+    public static final PneumaticsBase pneumaticsBase = PneumaticsBase.getForType(0, PneumaticsModuleType.CTREPCM);
```

Comment:
You should use the [`PneumaticsControlModule`](https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/wpilibj/PneumaticsControlModule.html) class directly.